site/src/github.rs

```diff
@@ -631,12 +631,15 @@ pub async fn post_finished(data: &InputData) {
                 "https://perf.rust-lang.org/compare.html?start={}&end={}",
                 commit.parent_sha, commit.sha
             );
+            let summary = categorize_benchmark(&commit, data).await;
             post_comment(
                 &data.config,
                 commit.pr,
                 format!(
                     "Finished benchmarking try commit ({}): [comparison url]({}).
 
+**Summary**: {}
+
 Benchmarking this pull request likely means that it is \
 perf-sensitive, so we're automatically marking it as not fit \
 for rolling up. Please note that if the perf results are \
```

Comment:
Just an idea: if the perf results don't return any significant perf changes, is this `rollup=never` necessary?

Reply:
A perf run is likely done when a change to a perf sensitive part of the compiler is done. It may not result in any significant perf changes when the perf run is done, but due to interaction with other PR's it may still have a perf change when actually landing.

Reply:
This categorization is also way too insensitive IMO. A perf run with improvements up to 0.6% and a couple of regressions up to 0.8% was marked as not having any significant changes. Many changes >0.2% across multiple benchmarks are unlikely to be noise. https://github.com/rust-lang/rust/pull/86404#issuecomment-866714363

![image](https://user-images.githubusercontent.com/17426603/123085656-ca1ff280-d422-11eb-9136-f14191e7be4b.png)

Reply:
I do think we can fine-tune sensitivity, but until we're even more confident I agree that we shouldn't drop the rollup=never. Our queue doesn't suffer much from it today anyway, to my knowledge.